scripts/setup.sh

```
@@ -91,30 +111,30 @@ fi
 
 # ---- Set up the backend ----
 echo -e "${YELLOW}Setting up backend...${NC}"
-cd ../backend
+cd "$REPO_ROOT/backend"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)